src/components/navbar/navbar.component.tsx

```diff
@@ -55,6 +57,20 @@ const Navbar: React.FC<NavbarProps> = ({
     }
   };
 
+  const [switcherToggler, setSwitcherToggler] = React.useState(false);
+
+  const Icon: React.FC<any> = () => {
```

Comment:
No. You should *only* have `Icon = switcherToggler ? Close20 : AppSwitcher20`.

Right now you define a new component (and you do that on every render). Also what's up with that style? Why is there a background color hardcoded in there?

Reply:
(Additionally that code is wrong, too, as no props are forwarded...).